WooCommerce/Classes/ViewRelated/Products/Edit Product/ProductFormViewController.swift

```diff
@@ -329,6 +340,7 @@ private extension ProductFormViewController {
 private extension ProductFormViewController {
     func editShippingSettings() {
         let shippingSettingsViewController = ProductShippingSettingsViewController(product: product) { [weak self] (weight, dimensions, shippingClass) in
+            ServiceLocator.analytics.track(.productShippingSettingsDoneButtonTapped)
```

Comment:
I'm curious why you decided to place these in `ProductFormViewController` and not in `ProductShippingSettingsViewController` itself? 🤔 

For me, it makes more sense for this to be inside `ProductShippingSettingsViewController` for the, albeit unlikely, case that we use `ProductShippingSettingsViewController` somewhere else.